DigitalLearningSolutions.Data.Migrations/202205061441_SeparateCentreEmailsTable.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace DigitalLearningSolutions.Data.Migrations
+{
+    using FluentMigrator;
+
+    [Migration(202205061441)]
+    public class SeparateCentreEmailsTable : Migration
+    {
+        public override void Up()

```

Comment:
Sorry, Jon's pointed out to me, it would be good to have uniqueness constraints on the UserCenterDetails table, on (userId, centreId) and (centreId, email[when not null] ) - see https://stackoverflow.com/a/767702 for the latter. I'll add this to the ticket.